docker/nodejs/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM nginx:latest
+
+########### Nginx Configuration starts here ###########
```

Comment:
Yeah, that's the plan. The PR is not fully ready. I pushed so that I don't loose the code and get comments from others.